examples/gno.land/p/demo/grc/grc721/igrc721_royalty.gno

```diff
@@ -0,0 +1,18 @@
+package grc721
+
+import (
+	"std"
+)
+
+// IGRC2981 follows the Ethereum standard 
+type IGRC2981 interface {	
+	// RoyaltyInfo retrieves royalty information for a tokenID and salePrice.
+	// It returns the payment address, royalty amount, and an error if any.
+	RoyaltyInfo(tokenID TokenID, salePrice uint64) (std.Address, uint64, error)
+}
+
+// RoyaltyInfo represents royalty information for a token.
+type RoyaltyInfo struct {
+	PaymentAddress std.Address // PaymentAddress is the address where royalty payment should be sent.
+	Percentage     uint64      // Percentage is the royalty percentage.
```

Comment:
Nit: add an example value next to the comment here for full clarity, ie `Percentage = 10` => `10%`

Reply:
[9067b07f2e89ce0bac176fecb8648086fcea23f3](https://github.com/linhpn99/gno/commit/9067b07f2e89ce0bac176fecb8648086fcea23f3)